src/main/java/org/fundacionjala/coding/fernandokatas/SortTheInnerContent.java

```diff
@@ -0,0 +1,45 @@
+package org.fundacionjala.coding.fernandokatas;
+
+import java.util.Arrays;
+
+public final class SortTheInnerContent {
+    private SortTheInnerContent() {
+    }
+
+    public static String sortTheInnerContent(final String words) {
+        StringBuilder result = new StringBuilder();
+        String[] sortWord = words.split(" ");
+        for (int i = 0; i < sortWord.length; i++) {
+            revertInnerWord(sortWord[i], result);
+            if (i < sortWord.length - 1) {
+                result.append(" ");
```

Comment:
Instead of appending blank space string 
You can use a StringJoiner class with delimiter " "